packages/contracts/contracts/TellerV2.sol

```diff
@@ -213,6 +222,29 @@ contract TellerV2 is
         collateralManager = ICollateralManager(_collateralManager);
     }
 
+    function setLenderManager(address _lenderManager)
+        public
+        reinitializer(CURRENT_CODE_VERSION)
```

Comment:
well need to increase this variable - also I'm thinking we should hard code these values in the modifier cause by increasing this variable value it would technically mean that someone could call `setCollateralManager` instead of `setLenderManager` as they both have the same modifier with the same updated version.

Reply:
i dont 100% understand

Reply:
if we deploy this contract, we or anyone could call `setCollateralManager` OR `setLenderManager` since the `reinitializer` modifier is using the same variable. we only want to allow updating the `LenderManager`. 

perhaps we just remove the `setCollateralManager` function as we don't need that function any more

Reply:
okay done.